server/sessionstore.go

```diff
@@ -74,9 +74,8 @@ func (ss *SessionStore) NewSession(conn interface{}, sid string) (*Session, int)
 		s.stop = make(chan interface{}, 1)                 // Buffered by 1 just to make it non-blocking
 		s.detach = make(chan string, 64)                   // buffered
 
-		if s.proto == CLUSTER {
-			s.remoteSessions = make(map[string]*remoteSession)
-		}
+		s.bkgTimer = time.NewTimer(time.Hour)
```

Comment:
I thought about it too, but I don't see how to avoid creating it:

https://github.com/tinode/chat/blob/tproxy/server/hdl_websock.go#L93
```
select {
	...
	case <-sess.bkgTimer.C:
	...
}
```
It can't be nil.